contrib/checkbox-provider-ce-oem/bin/serial_console_test.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import serial
+import serial.rs485
+import logging
+import sys
+import os
+from argparse import ArgumentParser
+
+formatter = logging.Formatter(
+    "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+)
+handler = logging.StreamHandler(sys.stdout)
+handler.setFormatter(formatter)
+logger = logging.getLogger("serial-test")
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
+
+
+def test_serial_console(serial_device, baudrate):
+    """Test the serial port when it is in console mode"""
+    try:
+        # Open serial port
+        with serial.Serial(serial_device, baudrate, timeout=1) as ser:
+            logger.info("Serial port opened successfully.")
+
+            # Send 'Enter Key'
+            logger.info("Sending 'Enter Key'...")
+            ser.write(os.linesep.encode())
+            response = ser.read(size=128).decode()
+            logger.info("Received response:\n'{}'".format(response))
```

Comment:
please use the correct way to log data with variable.
https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/logging-fstring-interpolation.html

Reply:
Done.